plugins/wake/README.md

```diff
@@ -0,0 +1,57 @@
+# wakeonlan
+
+This plugin provides a wrapper around the "wakeonlan" tool available from most
+distributions' package repositories, or from [the following website](https://github.com/jpoliv/wakeonlan).
```

Comment:
```suggestion
distributions' package repositories, or from [this website](https://github.com/jpoliv/wakeonlan).
```

It says "the following", but the link doesn't follow this text, but the text itself is linked.